Makefile

```diff
@@ -83,3 +83,16 @@ dist: clean ## builds source and wheel package
 
 install: clean ## install the package to the active Python's site-packages
 	python setup.py install
+
+
+test-prepare: ## Step up db containers needed for tests
+	opencdms-test-data startdb --containers opencdms-db
+	
+	sleep 20
```

Comment:
Is it likely that start up may take more than 20 seconds in a new environment where none of the images have been pulled before?

I'm wondering if we could use `pytest-docker-tools` to delay the tests until the containers are confirmed to be running.... but it may require bypassing the new CLI and using the docker compose file directly. However, that would be possible now that the `docker-compose.yml` is installed with the `opencdms-test-data` package (getting the path from `opencdms_test_data.__file__`)?

There's an example in the docs for setting the location of `docker-compose.yml` - see [here](https://pypi.org/project/pytest-docker/)